zfit_physics/models/pdf_cruijff.py

```diff
@@ -0,0 +1,100 @@
+from typing import Optional
+
+import zfit
+from zfit import z
+from zfit.util import ztyping
+from zfit.z import numpy as znp
+
+
+@z.function(wraps="tensor")
+def cruijff_pdf_func(x, mu, sigmal, alphal, sigmar, alphar):
+    """Caltulate the Cruijff PDF.
+
+    Args:
+        x: value(s) for which the PDF will be calculated.
+        mu: Mean value
+        sigmal: Left width parameter.
+        alphal: Left tail acceleration parameter.
+        sigmar: Right width parameter.
+        alphar: Right tail acceleration parameter.
+
+    Returns:
+        `tf.Tensor`: The calculated PDF values.
+
+    Notes:
+        Implementation from https://arxiv.org/abs/1005.4087
+    """
+    cond = znp.less(x, mu)
+
+    func = znp.where(
+        cond,
+        znp.exp(-0.5 * znp.square((x - mu) / sigmal) / (1 + alphal * znp.square((x - mu) / sigmal))),
+        znp.exp(-0.5 * znp.square((x - mu) / sigmar) / (1 + alphar * znp.square((x - mu) / sigmar))),
+    )
+    return func
+
+
+class Cruijff(zfit.pdf.BasePDF):
+    _N_OBS = 1
+
+    def __init__(
+        self,
+        mu: ztyping.ParamTypeInput,
+        sigmal: ztyping.ParamTypeInput,
+        alphal: ztyping.ParamTypeInput,
+        sigmar: ztyping.ParamTypeInput,
+        alphar: ztyping.ParamTypeInput,
+        obs: ztyping.ObsTypeInput,
+        *,
+        extended: Optional[ztyping.ExtendedInputType] = False,
+        norm: Optional[ztyping.NormInputType] = None,
+        name: str = "Cruijff",
+    ):
+        """Cruijff PDF.
+
+        Implementation from https://arxiv.org/abs/1005.4087
```

Comment:
```suggestion
        Implementation from https://arxiv.org/abs/1005.4087, citation 22, and http://hdl.handle.net/1794/9022, Equation IV.3
```
I would suggest to use this reference, as it's nicely written down, the Belle II papers have it as a citation point? Weirdest thing I've ever seen. But we can keep both